internal/statetransition/state_transition.go

```diff
@@ -2423,6 +2417,7 @@ func (a *Accumulator) SequentialDelta(
 		}
 	}
 
+	log.Println("maxReports", maxReports)
```

Comment:
Is this meant to be kept, or was it just for debugging?

Reply:
already fixed it :D